build.sbt

```diff
@@ -45,15 +45,16 @@ lazy val commonDependencies = Seq(
 lazy val buildSettings = Seq(
   organization := "com.snowplowanalytics",
   name := "snowplow-stream-collector",
-  version := "2.0.1",
+  version := "2.1.0-rc1",
   description := "Scala Stream Collector for Snowplow raw events",
   scalaVersion := "2.12.10",
+  javacOptions := Seq("-source", "11", "-target", "11"),
   resolvers ++= Dependencies.resolutionRepos
 )
 
 lazy val dockerSettings = Seq(
   maintainer in Docker := "Snowplow Analytics Ltd. <support@snowplowanalytics.com>",
-  dockerBaseImage := "snowplow-docker-registry.bintray.io/snowplow/base-debian:0.1.0",
+  dockerBaseImage := "snowplow-docker-registry.bintray.io/snowplow/base-debian:0.1.1",
```

Comment:
I think `base-debian` is at 0.2.1 already. @oguzhanunlu can you remind us what's the difference?

Reply:
[`snowplow/base-debian:0.2.2`](https://hub.docker.com/layers/snowplow/base-debian/0.2.2/images/sha256-ebe1ed4340aff0c4ae4c2d8fa8092c080b47aef222826ab52eb1f71f8de8cdee?context=explore) is available.

base-debian 0.1.x and 0.2.x images are based on JDK8 and JDK11 respectively. Given that collector targets JDK11 now, base image should be bumped to `0.2.2` here